.gitignore

```diff
@@ -9,7 +14,6 @@
 /celery
 /config/hydroshare-config.sh
 /config/hydroshare-config.yaml
-/hydroshare/local_settings.py
 /hydroshare/static
```

Comment:
Just to triple check: keep an eye on this and ensure it doesn't negatively impact the build system in Jenkins